src/status_im/contexts/wallet/account/view.cljs

```diff
@@ -19,7 +19,7 @@
            {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
            {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}]
     (not watch-only?) (conj {:id :dapps :label (i18n/label :t/dapps) :accessibility-label :dapps})
-    true              (conj {:id :about :label (i18n/label :t/about) :accessibility-label :about})))
+    :always           (conj {:id :about :label (i18n/label :t/about) :accessibility-label :about})))
```

Comment:
This is part of a `cond->` although it behaves the same, the Clojure convention is to use the keyword `:always`